site/src/pages/DeploymentSettingsPage/IdpOrgSyncPage/IdpOrgSyncPageView.tsx

```diff
@@ -0,0 +1,354 @@
+								<div>&nbsp;</div>
+								<Button
+									className="mb-px"
+									disabled={!idpOrgName || coderOrgs.length === 0}
+									onClick={async () => {
+										const newSyncSettings = {
+											...form.values,
+											mapping: {
+												...form.values.mapping,
+												[idpOrgName]: coderOrgs.map((org) => org.value),
+											},
+										};
+										void form.setFieldValue("mapping", newSyncSettings.mapping);
+										form.handleSubmit();
+										setIdpOrgName("");
+										setCoderOrgs([]);
+									}}
+								>
+									<Plus size={14} />
+									Add IdP organization
+								</Button>
+							</div>
+						</div>
+						<IdpMappingTable isEmpty={organizationMappingCount === 0}>
+							{form.values.mapping &&
+								Object.entries(form.values.mapping)
+									.sort()
+									.map(([idpOrg, organizations]) => (
+										<OrganizationRow
+											key={idpOrg}
+											idpOrg={idpOrg}
+											coderOrgs={getOrgNames(organizations)}
+											onDelete={handleDelete}
+										/>
+									))}
+						</IdpMappingTable>
+					</div>
+				</fieldset>
+			</form>
+		</div>
+	);
+};
+
+interface IdpMappingTableProps {
+	isEmpty: boolean;
+	children: React.ReactNode;
+}
+
+const IdpMappingTable: FC<IdpMappingTableProps> = ({ isEmpty, children }) => {
+	return (
+		<TableContainer>
+			<Table>
+				<TableHead>
+					<TableRow>
+						<TableCell width="45%">IdP organization</TableCell>
+						<TableCell width="55%">Coder organization</TableCell>
+						<TableCell width="10%" />
+					</TableRow>
+				</TableHead>
+				<TableBody>
+					<ChooseOne>
+						<Cond condition={isEmpty}>
+							<TableRow>
+								<TableCell colSpan={999}>
+									<EmptyState
+										message={"No organization mappings"}
+										isCompact
+										cta={
+											<Button variant="outline" asChild>
+												<a
+													href={docs("/admin/users/idp-sync")}
+													className="no-underline"
+												>
+													<SquareArrowOutUpRight size={14} />
+													How to set up IdP organization sync
+												</a>
+											</Button>
+										}
+									/>
+								</TableCell>
+							</TableRow>
+						</Cond>
+
+						<Cond>{children}</Cond>
+					</ChooseOne>
+				</TableBody>
+			</Table>
+		</TableContainer>
+	);
+};
+
+interface OrganizationRowProps {
+	idpOrg: string;
+	coderOrgs: readonly string[];
+	onDelete: (idpOrg: string) => void;
+}
+
+const OrganizationRow: FC<OrganizationRowProps> = ({
+	idpOrg,
+	coderOrgs,
+	onDelete,
+}) => {
+	return (
+		<TableRow data-testid={`group-${idpOrg}`}>
+			<TableCell>{idpOrg}</TableCell>
+			<TableCell>
+				<OrganizationPills organizations={coderOrgs} />
+			</TableCell>
+			<TableCell>
+				<Button
+					variant="outline"
+					className="w-8 h-8 px-1.5 py-1.5 text-content-secondary"
+					onClick={() => onDelete(idpOrg)}
+				>
+					<Trash />
+					<VisuallyHidden.Root>Delete IdP mapping</VisuallyHidden.Root>
```

Comment:
I was playing around with Tailwind over the holiday, and I saw that they have a visually hidden class via `sr-only` (screen reader only).

I looked at Radix's source code, and their VisuallyHidden is basically no different from this:
```tsx
<span className="sr-only">Delete IdP mapping</span>
```
I think I'd prefer to do that rather than add more abstraction via Radix's base Primitive component